### PR TITLE
new langKnown example + bib ref

### PR DESCRIPTION
**Pull request by stuartyeates**
The thing I'm least confident about is the bibl; bibls have always confused me and I've never see a bibl pointing to a file on github.

### CODE REVIEW

**Comment by sydb** [comment]
With respect to a bibliographic entry referring to an entry in a GitHub file, I hear you. I am not 100% sure how to do this, either. In one sense, the “right” way to encode this is
~~~xml 
        <biblFull source="https://raw.githubusercontent.com/iljackb/Mixtepec_Mixtec/master/MIX-People.xml">
          <titleStmt>
            <title>Mixtepec-Mixtec Project Personography</title>
            <principal>Jack Bowers</principal>
          </titleStmt>
          <publicationStmt>
            <authority>
              <name>Jack Bowers</name>
              <name>Jeremías Salazar</name>
              <name>Tisu'ma Salazar</name>
            </authority>
            <availability>
              <licence>CC-BY</licence>
            </availability>
          </publicationStmt>
          <sourceDesc>
            <p>Information about the source</p>
          </sourceDesc>
        </biblFull>
~~~
After all, that is what `<biblFull>` is for, at least IMHO. But a) some useful information is missing from Jack’s `<titleStmt>`, and b) I really doubt the TEI Stylesheets will be able to handle this. Some would say that structured information like this is really intended for search, retrieval, and access, not formatting-for-display. Something as simple as
~~~xml
        <bibl>
          <author>Bowers, Jack</author>
          <title>Mixtepec-Mixtec Project Personography</title>
          <ptr target="https://raw.githubusercontent.com/iljackb/Mixtepec_Mixtec/master/MIX-People.xml#element(JS/6)"/>
        </bibl>
~~~
does the trick technically, but a lot of folks (and a lot of software) will not know what to make of the XPointer. Besides, best practices is to include the date of access. Thus:
~~~xml
        <bibl>
          <author>Bowers, Jack</author>
          <title>Mixtepec-Mixtec Project Personography</title>
          <ptr target="https://raw.githubusercontent.com/iljackb/Mixtepec_Mixtec/master/MIX-People.xml#element(JS/6)"/>
          <date type="accessed" when="2021-08-20"/>
        </bibl>
~~~
While it is traditional (and appropriate) to include the publisher, in this case one is not listed, just authorities, and there are 3 of them. (I take the distributor to be GitHub, but that is not very interesting, and evident from the URL anyway.)

File: P5/Source/Guidelines/en/BIB-Bibliography.xml [hunk 1]
```diff
@@ -116,6 +116,14 @@ $Id$
       In <editor role="tr">James E. Irby</editor> tr. <title level="m">Labyrinths: Selected Stories and Other
       Writings</title>, <pubPlace>New York</pubPlace>: <publisher>New Directions</publisher>, (<date>1962</date>),
       <biblScope unit="pp">pp.202-203</biblScope>.</bibl>
+      <bibl xml:id="github-mix-mix">
```
Comment (sydb) [inline P5/Source/Guidelines/en/BIB-Bibliography.xml:119]: See comment on main page.

File: P5/Source/Specs/langKnown.xml [hunk 2]
```diff
@@ -63,6 +63,18 @@
       <langKnown tag="fr" level="M">French</langKnown>
     </egXML>
   </exemplum>
+  <exemplum xml:lang="en">
+    <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#github-mix-mix">
+      <person sex="m" role="speaker, collaborator">
```
Comment (sydb) [inline P5/Source/Specs/langKnown.xml:68]: Need to remove the comma. (See [issue](https://github.com/iljackb/Mixtepec_Mixtec/issues/111)).

**Review by sydb — CHANGES_REQUESTED** [review]

**Comment by sydb** [comment]
[Meant to put this as the main comment for my review, but forgot to, and don’t know how to do so after-the-fact.]

I do not know what is going on with the Travis build failing, and would not worry about it. All builds that work on my local system (all but NVDL validity and PDF output, IIRC) worked fine.

**Comment by sydb** [comment]
Note: this PR arose from an existing [issue](#2139).

**Comment by ebeshero** [comment]
@stuartyeates Can you update your repo with changes from our dev branch and make the small change requested here? We were noticing that the tests are failing *likely* because the branch is out of date. If this doesn't work we can copy the work to a new branch.

**Pull request closed without merging**